tests/behat/behat_mod_coursework.php

```diff
@@ -1889,7 +2025,7 @@ public function thereIsFinalFeedback() {
     /**
      * @Given /^there is final feedback from the other teacher$/
      */
-    public function thereIsFinalFeedbackFromTheOtherTeacher() {
+    public function there_is_final_feedbackFromTheOtherTeacher() {
```

Comment:
there's a few like this - are we trying to get rid of CamelCase? 
but we keep it when adding _ would lead to too many characters?